recipes/ampform-dpd/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "ampform-dpd" %}
+{% set version = "0.1.0" %}
```

Comment:
Is there a reason to be starting with this old version that looks like it was released in Dec 2022?

Reply:
That's reasonable if that's what you want to do, although you're not required to.